composer.json

```diff
@@ -158,7 +158,7 @@
         "symfony/dom-crawler": "^6.2",
         "ycloudyusa/useless_machines": "dev-main",
         "ycloudyusa/y_lb": ">=2.5.1, <2.6.0",
-        "ycloudyusa/y_lb_demo_content": "^1 || dev-main",
+        "ycloudyusa/y_lb_demo_content": "v2.x-dev",
```

Comment:
`v2` is not going to be the final branch name, right?

Reply:
sure, it's going to be release tag with ^Caret @froboy